self-api/src/main/java/com/selfxdsd/api/Language.java

```diff
@@ -56,6 +65,7 @@ public Language(
         final String commandsFileName,
         final String responsesFileName
     ) {
+        this(new Properties(), new Properties());
```

Comment:
@criske Why is this needed? :D 

I would prefer to leave the constructor as it was initially.

Reply:
@amihaiemil Needed to test `file://` protocol, that works only with absolute paths (I couldn't make it to work otherwise).
Just adding a file full path in `responses.properties` will break the ci though.